From 69bbdb7e26100f707402be63b7d8dcb69d78c597 Mon Sep 17 00:00:00 2001
From: David Drysdale <drysdale@google.com>
Date: Thu, 2 Oct 2014 12:02:08 +0100
Subject: [PATCH 4/9] chaps: Downgrade excessive logs

Logs appear on stdout/stderr on Linux.

BUG=None
TESTED=Chaps unit tests plus PKCS11 tests
---
 chaps/chaps_proxy.cc   | 2 +-
 chaps/isolate_linux.cc | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/chaps/chaps_proxy.cc b/chaps/chaps_proxy.cc
index 42163a3..f950234 100644
--- a/chaps/chaps_proxy.cc
+++ b/chaps/chaps_proxy.cc
@@ -38,7 +38,7 @@ bool ChapsProxyImpl::Init() {
     if (proxy_.get()) {
       if (!WaitForService())
         return false;
-      LOG(INFO) << "Chaps proxy initialized (" << kChapsServicePath << ").";
+      VLOG(1) << "Chaps proxy initialized (" << kChapsServicePath << ").";
       return true;
     }
   } catch (DBus::Error err) {
diff --git a/chaps/isolate_linux.cc b/chaps/isolate_linux.cc
index 6be57c9..e91b725 100644
--- a/chaps/isolate_linux.cc
+++ b/chaps/isolate_linux.cc
@@ -60,7 +60,7 @@ bool IsolateCredentialManager::GetUserIsolateCredential(
   const FilePath credential_file = FilePath(kIsolateFilePath).Append(user);
   if (!base::PathExists(credential_file) ||
       !base::ReadFileToString(credential_file, &credential_string)) {
-    LOG(INFO) << "Failed to find or read isolate credential for user "
+    VLOG(1) << "Failed to find or read isolate credential for user "
                << user;
     return false;
   }
-- 
1.9.1

